OpenSim/Sandbox/MatlabScripts/matlabArrayToOpenSimTimeSeriesTable.m

```diff
@@ -0,0 +1,66 @@
+function table = matlabArrayToOpenSimTimeSeriesTableVec3(table_array)
+%% Matlab Utility to take a Matlab Struct and convert into an OpenSim Table
+
+
+%%
+import org.opensim.modeling.*
+% table_array = force_array
+
+%% make a empty table
+table = DataTable;
+
+%% get the label names from the structure
+labels = fieldnames(table_array);    
+
+%% Set the number of columns and the column labels
+colnames = StdVectorString;
+for i = 1 : length(labels)
+   
+    if ~strcmp(labels{i},'time')
+        colnames.add(labels{i})
+    end
+end
+
+table.setColumnLabels(colnames);
+
+%% Create an empty Array
+row = RowVector(colnames.size, 0);
+
+for ii  = 1 : length(table_array.(labels{1}))
+
+    table.appendRow(0, row)
+    
+end
+    
+%% write values to the OpenSim table
+for i  = 0 : length(labels) -1
+
+    if strcmp(labels(i+1),'time')
```

Comment:
Is there a reason why parentheses are used here, but curly braces are used on line 44?